appcues/src/main/java/com/appcues/ElementTargeting.kt

```diff
@@ -0,0 +1,133 @@
+package com.appcues
+
+import android.view.View
+import com.squareup.moshi.JsonClass
+import java.util.UUID
+
+/**
+ * Defines an element targeting strategy type, which can be used to capture view layout
+ * information in the application UI as well as render element targeted experiences.
+ */
+interface ElementTargeting {
```

Comment:
the commends define well that this is supposed to do. wondering if the naming here could include "strategy" to make it more clear, how does ElementTargeting sounds without any context do you think?

Reply:
Android could be named AndroidTargetingStrategy, etc